### PR TITLE
Add submodule for post-processing functions

### PR DESCRIPTION
**Pull request by lkluft**

### CODE REVIEW

File: orcestra/postprocess/level0.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    return ds.rename({"tid": "time", "TIME": "time"}).set_index(time="time")
+
+
+def _bahamas_line_of_sight(ds):
+    """Calculate the length of line-of-sight (LOS)."""
+    return ds.assign(
+        LOS=lambda dx: dx.IRS_ALT
+        / np.cos(np.radians(dx.IRS_THE))
+        / np.cos(np.radians(dx.IRS_PHI))
+    )
```
Comment (d70-t) [inline orcestra/postprocess/level0.py:15]: The `LOS` variable seems to assume a certain sensor viewing direction (e.g. the radar), thus is not really a variable which should generically be associated with the basic instrumentation system. Instead, this variable likely should be attached to the instrument it's referring to.

File: orcestra/postprocess/level0.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def bahamas(ds):
+    """Post-processing of BAHAMAS datasets."""
+    return ds.pipe(
+        _bahamas_fix_time,
+    ).pipe(
+        _bahamas_line_of_sight,
+    )
```
Comment (d70-t) [inline orcestra/postprocess/level0.py:24]: (see comment above)

File: orcestra/postprocess/level0.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def _radar_ql_fix_time(ds):
+    """Fix time coordinate of RADAR QL datasets."""
+    datetime = np.datetime64("1970-01-01", "ns")
+    datetime += ds.time.values * np.timedelta64(1, "s")
+    datetime += ds.microsec.values * np.timedelta64(1, "us")
```
Comment (d70-t) [inline orcestra/postprocess/level0.py:31]: I'd generally recommend not to modify variables, but rather to assign them once. In this case, e.g.:
```suggestion
    datetime = np.datetime64("1970-01-01", "ns") + ds.time.values * np.timedelta64(1, "s") + ds.microsec.values * np.timedelta64(1, "us")
```
(maybe in multiple lines)

**Review by d70-t — COMMENTED** [review]
Thanks for starting the `postprocess` sub-package. Apart from the `LOS` variable, I agree and like it very much.

File: orcestra/postprocess/level0.py (hunk 3 above)
Comment (lkluft) [reply orcestra/postprocess/level0.py:31]: Good catch! I missed that this modifies the `ds.time` array

File: orcestra/postprocess/level0.py (hunk 1 above)
Comment (lkluft) [reply orcestra/postprocess/level0.py:15]: I am fine with dropping this processing step

**Pull request merged**